bpfmt/bpfmt.go

```diff
@@ -40,17 +41,18 @@ func usage() {
 	os.Exit(2)
 }
 
-// If in == nil, the source is the contents of the file with the given filename.
-func processFile(filename string, in io.Reader, out io.Writer) error {
-	if in == nil {
-		f, err := os.Open(filename)
-		if err != nil {
-			return err
-		}
-		defer f.Close()
-		in = f
+func processFile(filename string, out io.Writer) error {
+	f, err := os.Open(filename)
+	if err != nil {
+		return err
 	}
+	defer f.Close()
+	in := f
```

Comment:
This doesn't make sense any more, either rename f to in or pass f to processReader

Reply:
thanks